var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -382,7 +382,7 @@ def configure_args(self):
                 config_args.append('--enable-contrib-no-build=vt')
 
         # Multithreading support
-        if spec.satisfies('@1.5.4:'):
+        if spec.satisfies('@1.5.4:1.999'):
```

Comment:
Can you please double check if `1.999` is correct or if it should be `2.999` like they suggest [here](https://github.com/open-mpi/ompi/issues/4435#issuecomment-341428434)?

Reply:
Howard Pritchard agrees with you. Thanks for the cross check. Correct package.py pushed.

Reply:
@floquet I don't see any change here. Did you push to another branch?